Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -168,6 +168,20 @@ public FieldReference GetConstantPoolReference(string stringConstant)
         return field;
     }
 
+    public FieldReference GetConstantPoolReference(byte[] dataConstant)
+    {
+        var hash = dataConstant.GetHashCode().ToString();
```

Comment:
It's not a good idea to take a hash code of a runtime array. At least this won't get you a reproducible build, and at the worst, it will outright break.

Probably you wanted to get a hash code from the array's contents. And also, mind that the runtime's hash code implementation is never meant to be stable, and may change its results between runs, which will, again, break the reproducibility of builds.

I've written a suggested implementation using SHA256 here when I recognized that we should actually generify the `_stringConstantHolders` to use a byte array as the key (using an appropriate comparer, of course). And then, rename all the related stuff accordingly ("StringConstant" → "DataConstant").